### DISCUSSION TITLE
Fibaro FGFS101 Creates Basic Switch

### DISCUSSION THREAD

**Discussion by dcmeglio** [zwave-js/zwave-js#6596] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
Fibaro FGFS101 (a leak sensor) has a Basic switch after the recent changes. I tried doing the proposed re-interview fix, and now I have two Basic switches:
![image](https://github.com/zwave-js/node-zwave-js/assets/21957250/fcbf0dd3-4470-4672-8d8a-77d64e837acb)
Oddly doing it a 3rd time did not create a 3rd switch

What did you expect to happen instead?
This switch doesn't seem to do anything or make any sense for this device.

Pair a FGFS101 and look at the device entities in Home Assistant (Note I know this is about Home Assistant but https://community.home-assistant.io/t/psa-upcoming-z-wave-js-changes-and-z-wave-certification-how-it-impacts-you/638288#what-probably-brought-you-here-basic-cc-light-3 says to report it here)

https://github.com/zwave-js/node-zwave-js/issues/6516 seems to suggest this bug is fixed but I can't get the switch to disappear with a re-interview.

### Software versions

Driver (node-zwave-js): NA

Z-Wave JS UI: 3.1.0

Home Assistant Z-Wave Integration: NA

Home Assistant Z-Wave JS Addon: 3.1.0

ioBroker.zwave2 Adapter: NA

If you are using something non-standard, tell us here: ...

### Device information

Manufacturer: Fibaro
Model name: FGFS101 
Node ID: 87

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[z-ui_2024-01-06.log](https://github.com/zwave-js/node-zwave-js/files/13867359/z-ui_2024-01-06.log)

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment]
What the bot said.
